benchexec/resources.py

```diff
@@ -15,191 +10,157 @@
 import math
 import os
 import sys
+from functools import cmp_to_key
```

Comment:
I would suggest to just import `functools`, it seems not worth to import `cmp_to_key` directly into our namespace because we use it only once. And please put the import in alphabetical order.